src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -436,12 +436,18 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
                 headerName: 'Created By',
                 field: 'CreatedBy',
                 minWidth: 160,
-                valueFormatter: (cellParams) => cellParams.value && (cellParams.value.FirstName + ' ' + cellParams.value.LastName).trim(),
+                cellRenderer: (cellParams) => {
```

Comment:
We should create a common function for the `cellRenderer` and `keyCreator`.

Reply:
done